src/player.js

```diff
@@ -286,6 +299,33 @@ class Player extends FakeEventTarget {
 
   //  <editor-fold desc="Playback Interface">
   /**
+   * The player readiness
+   * @public
+   * @returns {Promise<*>} - The ready promise
+   */
+  ready(): Promise<*> {
+    return this._readyPromise ? this._readyPromise : Promise.resolve();
+  }
+
+  /**
+   * Load media
+   * @public
+   * @returns {void}
+   */
+  load(): void {
+    if (this._engine) {
+      this._engine.load().then((data) => {
+        this._tracks = data.tracks;
+        this.dispatchEvent(new FakeEvent(CustomEvents.TRACKS_CHANGED, {tracks: this._tracks}));
+      }).catch((error) => {
+        this.dispatchEvent(new FakeEvent(Html5Events.ERROR, error));
+      });
+    } else {
+      this._config.preload = 'auto';
```

Comment:
No need to handle cases where engine is not ready due to calling player without sources